data/docker-compose-data_management.yml

```diff
@@ -20,6 +20,22 @@ services:
         - API_POSTGRES_USER=marxan-api
         - API_POSTGRES_PASSWORD=marxan-api
         - API_POSTGRES_DB=marxan-api
+
+  marxan-restore-data:
+    build:
+        context: ./data_restore
+        dockerfile: Dockerfile
+    user: "$UID:$GID"
+    command: make
+    environment:
+        - POSTGRES_API_SERVICE_PORT=3432
+        - POSTGRES_GEO_SERVICE_PORT=3433
+        - GEO_POSTGRES_USER=marxan-geo-api
+        - GEO_POSTGRES_PASSWORD=marxan-geo-api
+        - GEO_POSTGRES_DB=marxan-geo-api
+        - API_POSTGRES_USER=marxan-api
+        - API_POSTGRES_PASSWORD=marxan-api
+        - API_POSTGRES_DB=marxan-api
```

Comment:
if we want to streamline this a bit (later), I would suggest to avoid hardcoding envvars here - why not use `env_file: '../.env'` here? These look like basically the same env vars we have there and if we need to override some, we could use a second envfile and the value for any given key from the latest file in the list will be the one to be used: https://docs.docker.com/compose/compose-file/compose-file-v3/#env_file